src/status.lambda.ts

```diff
@@ -155,9 +155,11 @@ exports.handler = async function (event: Partial<AWSLambda.APIGatewayProxyEvent>
 
   // setup url
   const setupToken = (await getSecretJsonValue(process.env.SETUP_SECRET_ARN)).token;
-  if (setupToken) {
+  if (setupToken && process.env.SETUP_FUNCTION_URL) {
     status.github.setup.status = 'Pending';
     status.github.setup.url = `${process.env.SETUP_FUNCTION_URL}?token=${setupToken}`;
+  } else if (process.env.SETUP_FUNCTION_URL) {
```

Comment:
Are you sure about this one? If the URL exists and the token doesn't, that should mean it's complete.